conduct/code_of_conduct.md

```diff
@@ -0,0 +1,93 @@
+
+# Project Jupyter Code of Conduct and Community Guidelines
+
+Project Jupyter is an engaged and respectful community made up of people
+from all over the world. Your involvement helps us to further our
+mission and to create an open platform that serves a broad range of
+communities, from research and education, to journalism, industry and
+beyond. This diversity is one of our biggest strengths, but it can also
+lead to communication issues and conflicts. To that end, we have a few
+ground rules we ask that our community members adhere to.
```

Comment:
> Any particular reason? To that end seems clearer to me...

I am not the native speaker. This one just a comment on the first pass on the text. "To that end" was unclear to me, since it is not specified what *end*, "that" refers to.